mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -447,23 +447,22 @@ impl S3CrtClientInner {
             .on_telemetry(move |metrics| {
                 let _guard = span_telemetry.enter();
 
-                let http_status = metrics.status_code().unwrap_or(-1);
-                let request_failure = !(200..299).contains(&http_status);
+                let http_status = metrics.status_code();
+                let request_failure = http_status.map(|status| !(200..299).contains(&status)).unwrap_or(true);
+                let crt_error = Some(metrics.error()).filter(|e| e.is_err());
                 let request_type = request_type_to_metrics_string(metrics.request_type());
                 let request_id = metrics.request_id().unwrap_or_else(|| "<unknown>".into());
                 let duration = metrics.total_duration();
                 let ttfb = metrics.time_to_first_byte();
                 let range = metrics.response_headers().and_then(|headers| extract_range_header(&headers));
 
-                let log_level = status_code_to_log_level(http_status);
-
                 let message = if request_failure {
-                    "request failed"
+                    "CRT request failed"
                 } else {
-                    "request finished"
+                    "CRT request finished"
                 };
-                event!(log_level, %request_type, http_status, ?range, ?duration, ?ttfb, %request_id, "{}", message);
-                trace!(detailed_metrics=?metrics, "request completed");
+                debug!(%request_type, ?crt_error, http_status, ?range, ?duration, ?ttfb, %request_id, "{}", message);
+                trace!(detailed_metrics=?metrics, "CRT request completed");
```

Comment:
it's just bikeshedding but i don't think "CRT request" is the right way to describe these. it's an S3 request that failed. if we're going to push these down to debug level, i don't see the need to also write "CRT" here.